fmlearlydisplay/src/test/java/net/minecraftforge/fml/earlydisplay/tests/DisplayTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.fml.earlydisplay.tests;
+
+import net.minecraftforge.fml.earlydisplay.*;
+import net.minecraftforge.fml.loading.progress.StartupNotificationManager;
+import org.junit.jupiter.api.Test;
+import org.lwjgl.opengl.GL;
+
+import java.util.List;
+
+import static org.lwjgl.glfw.GLFW.*;
+import static org.lwjgl.opengl.GL11.*;
+
+public class DisplayTest {
```

Comment:
This test class does nothing.
Please remove.

Reply:
It taught me how to write GL code. Does the fact that it's now completely useless mean it deserves to die??????????

Reply:
Yes